### PR TITLE
HTML and CSS Practice

### PR DESCRIPTION
**Pull request by AnthonyShostak**
### HTML and CSS Practice

[Demo](https://anthonyshostak.github.io/popup/) |
[Code base](https://github.com/AnthonyShostak/popup/)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by AnthonyShostak** [comment]
I have completed self-checks. Thank you!

File: submissions/AnthonyShostak/Hooli-popup-task/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    <li class="nav__item mail"><a href="#">Mail</a></li>
+                    <li class="nav__item images"><a href="#">Images</a></li>
+                    <li class="nav__item popup__nav__item">
+                        <input type="checkbox" id="popup" class="popup_checkbox">
+                        <label for="popup">
+                            <img src="popup-icons/popup-button.png" alt="popup-button" class="icon popup__icon">
+                        </label>
+                        <div class="popup">
+                            <ul class="popup__items">
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/cart.png" alt="cart" class="icon">Cart</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/AnthonyShostak/Hooli-popup-task/index.html:22]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/AnthonyShostak/Hooli-popup-task/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/games.png" alt="games" class="icon">Games</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/location.png" alt="location" class="icon">Location</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/mic.png" alt="microphone" class="icon">Microphone</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/night-mode.png" alt="night-mode" class="icon">Night mode</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/scan.png" alt="scan" class="icon">Scan</a></li>
+                                <li class="more__below">
+                                    <input type="checkbox" id="popup__more" class="checkbox__more">
+                                    <label for="popup__more" class="more">Press to see more</label>
+                                    <label for="popup__more" class="less">Press to see less</label>
```
Comment (A-Ostrovnyy) [inline submissions/AnthonyShostak/Hooli-popup-task/index.html:34]: Add `cursor: pointer` to indicate to users about an interactive element.
Logically when the second part of the popup is open this button should be hidden.

File: submissions/AnthonyShostak/Hooli-popup-task/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                                    </div>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                </ul>           
+            </nav>
+        </header>
+        <main class="main">
+        <footer></footer>
```
Comment (A-Ostrovnyy) [inline submissions/AnthonyShostak/Hooli-popup-task/index.html:50]: Avoid unused tags in your work.

File: submissions/AnthonyShostak/Hooli-popup-task/index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        </main>
+    </body>
+
+
```
Comment (A-Ostrovnyy) [inline submissions/AnthonyShostak/Hooli-popup-task/index.html:54]: Put an empty line at the end of every file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1). Tune your code editor's settings so it does this for you.

File: submissions/AnthonyShostak/Hooli-popup-task/index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        <header class="header">
+            <nav class="nav__header">
+                <ul class="nav__list">
+                    <li class="nav__item mail"><a href="#">Mail</a></li>
+                    <li class="nav__item images"><a href="#">Images</a></li>
+                    <li class="nav__item popup__nav__item">
+                        <input type="checkbox" id="popup" class="popup_checkbox">
+                        <label for="popup">
+                            <img src="popup-icons/popup-button.png" alt="popup-button" class="icon popup__icon">
+                        </label>
```
Comment (A-Ostrovnyy) [inline submissions/AnthonyShostak/Hooli-popup-task/index.html:19]: Add `cursor: pointer` to indicate to users about an interactive element.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/AnthonyShostak/Hooli-popup-task/index.html (hunk 1 above)
Comment (AnthonyShostak) [reply submissions/AnthonyShostak/Hooli-popup-task/index.html]: Removed unnecessary alt-texts.

File: submissions/AnthonyShostak/Hooli-popup-task/index.html (hunk 2 above)
Comment (AnthonyShostak) [reply submissions/AnthonyShostak/Hooli-popup-task/index.html]: Added cursor: pointer to interactive elements.
Removed "Press to see less" button.
Also "Press to see more" button is now hidden after it's clicked.

File: submissions/AnthonyShostak/Hooli-popup-task/index.html (hunk 3 above)
Comment (AnthonyShostak) [reply submissions/AnthonyShostak/Hooli-popup-task/index.html]: Removed unused tags.

File: submissions/AnthonyShostak/Hooli-popup-task/index.html (hunk 4 above)
Comment (AnthonyShostak) [reply submissions/AnthonyShostak/Hooli-popup-task/index.html]: Put empty lines by using "Insert Final Newline" parameter in VCS.

File: submissions/AnthonyShostak/Hooli-popup-task/index.html (hunk 5 above)
Comment (AnthonyShostak) [reply submissions/AnthonyShostak/Hooli-popup-task/index.html]: Add cursor: pointer to interactive elements.

File: submissions/AnthonyShostak/Hooli-popup-task/index.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    <body>
+        <header class="header">
+            <nav class="nav__header">
+                <ul class="nav__list">
+                    <li class="nav__item mail"><a href="#">Mail</a></li>
+                    <li class="nav__item images"><a href="#">Images</a></li>
+                    <li class="nav__item popup__nav__item">
+                        <input type="checkbox" id="popup" class="popup_checkbox">
+                        <label for="popup">
+                            <img src="popup-icons/popup-button" alt="popup-button" class="icon popup__icon">
```
Comment (A-Ostrovnyy) [inline submissions/AnthonyShostak/Hooli-popup-task/index.html:18]: Add extension to Image url.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Did you update your demo?)

File: submissions/AnthonyShostak/Hooli-popup-task/index.html (hunk 6 above)
Comment (AnthonyShostak) [reply submissions/AnthonyShostak/Hooli-popup-task/index.html:18]: I don't understand what kind of extension I have to add. Is something works wrong?

Comment (A-Ostrovnyy) [inline submissions/AnthonyShostak/Hooli-popup-task/index.html:18]:
```suggestion
                            <img src="popup-icons/popup-button.png" alt="popup-button" class="icon popup__icon">
```

**Review by A-Ostrovnyy — APPROVED** [review]
Don't forget to define the image format in url)
Well done!

**Pull request merged**